src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
-body {
+@import './variables';
+@import './body';
+@import './grade';
+@import './price';
+
+p {
   margin: 0;
 }
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__items {
+  object-fit: contain;
+  margin: 32px 19px 40px;
```

Comment:
The use of specific pixel values for margin like `margin: 32px 19px 40px;` can be rigid and might not scale well if the design changes. Consider using relative units like `rem` or `em`, or a combination of both fixed and relative units for better scalability and maintainability.